cfpb/consumerfinance.gov
# Deprecate custom Django admin logout template

When users logout from the Django admin (not to be confused with the totally separate action of logging out from the Wagtail admin) they see a custom template which is currently visually broken.

This commit removes that custom template and sets the Django setting [`LOGOUT_REDIRECT_URL`](https://docs.djangoproject.com/en/4.2/ref/settings/#logout-redirect-url) so that users log out and are taken back to the Wagtail login view.

|Current|Default Django logout view|With this change|
|-|-|-|
|<img width="1552" alt="image" src="https://user-images.githubusercontent.com/654645/231858859-f53dfd59-8e48-463f-9680-54af345a51e7.png">|<img width="1552" alt="image" src="https://user-images.githubusercontent.com/654645/231859030-9e5536ed-620d-4c09-9e22-e1f73b6b856b.png">|<img width="1552" alt="image" src="https://user-images.githubusercontent.com/654645/231859148-3378728d-0b5e-4141-a07f-d5da19b8b715.png">|

## How to test this PR

1. Sign into the Wagtail admin as a user with permissions to access the Django admin.
2. Click on "Django Admin" in the Wagtail sidebar.
3. Click on "Log out" in the upper right.
4. Note that you are logged out and redirected back to the Wagtail admin.

## Checklist
